source/NVDAObjects/window/excel.py

```diff
@@ -883,54 +899,18 @@ def _get_states(self):
 			states.add(controlTypes.STATE_PROTECTED)
 		return states
 
-	@scriptHandler.script(gestures=(

```

Comment:
Why you've removed these assigments and then rewritten them again below just changing formatting?

Reply:
Just daved some lines but I noticed that this throws lint error for whatever reason. Ichanged it back.